### PR TITLE
Add some more information based on recent frequently asked questions.

### PR DESCRIPTION
**Pull request by halkeye**

### CODE REVIEW

**Comment by halkeye** [comment]
@keywordnew do you also have the emoji-foo stuff for the other series? What is september? fall series?

**Comment by keywordnew** [comment]
I stuck a few emojis and month brackets in the [README.md](https://github.com/nodeschool/vancouver/blob/master/meetup-templates/README.md) for the templates directory.

We ended up using a different one for summer after all :)

File: meetup-templates/meetup-template-nodeschool.md [hunk 1]
```diff
@@ -18,10 +18,12 @@ A note from **Clio** about how to get there:
 
 Meetup description
 
-Are you a beginner at using Node.js and/or JavaScript? Are you comfortable with either, and just want to experiment and learn in the company of other nice humans?
+Are you a beginner at using Node.js and/or JavaScript? Are you wanting a refresher? Are you looking to level up your knowledge? 
+
+We don't have set classes, and as such it is considered a self study program. The Node School organization has a lot of workshops for you to learn from, and we help out along the way, but that doesn't mean you have to do the workshops, we are a diverse group of people learning and growing together.
```
Comment (keywordnew) [inline meetup-templates/meetup-template-nodeschool.md]: nit: name

```suggestion
We don't have set classes, and as such it is considered a self study program. The NodeSchool organization has a lot of workshops for you to learn from, and we help out along the way, but that doesn't mean you have to do the workshops, we are a diverse group of people learning and growing together.
```

**Review by keywordnew — COMMENTED** [review]
Thank you for the update. Tiny nit, aside from that, LGTM.

**Review by keywordnew — APPROVED** [review]
Pre-emptive! 👍🏽

**Pull request merged**